### PR TITLE
High level installation instructions

### PR DESCRIPTION
**Pull request by leftclickben**
I wrote some instructions for installing everything.  There wasn't an ideal place to put them so I've put them here in the `shop-theme` repo, which had a more-or-less empty `README.md` file.

### CODE REVIEW

**Comment by leftclickben** [comment]
I've set this to merge into `rwahs/RebaseOntoLuma`, but after it is merged, we should merge that into `rwahs/master` because that is the direction we have chosen to go in.

**Comment by leftclickben** [comment]
These instructions were based on what I did to get it working in both staging and UAT, but have not been independently tested, so they might not be perfect.  I don't know if it's worth spinning up a micro instance just to test the instructions?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,123 @@
...
+cd /srv/deploy
+git clone https://github.com/rwahs/shop-theme.git
+git clone https://github.com/rwahs/shop-migration.git
+```
+
+### 2. Install Magento code
+
+* Version: 2.0.x
+* Method: composer
+
```
Comment (kehh) [inline README.md:60]: You also need an access key as defined at http://devdocs.magento.com/guides/v2.0/install-gde/prereq/connect-auth.html

File: README.md [hunk 2]
```diff
@@ -1,2 +1,127 @@
...
+ 
+* http://devdocs.magento.com/guides/v2.0/install-gde/prereq/connect-auth.html
+
+### 3. Run the Magento installer
+
+This step is required to configure the database connection and backend URL; other settings will be overwritten by 
+installing the database seed in the next step.
+
+```
+cd /srv/sites/documentation-shop.histwest.org.au
```
Comment (kehh) [inline README.md]: set permissions on bin/magento (and other bits as well)

**Comment by kehh** [comment]
so anyway:

``` bash
cp -r ../website-theme app/design/frontend/Gaia/rwahsluma
```

Didn't seem to work.

**Comment by kehh** [comment]
and then I kicked myself many times when I realised my mistake!!!

**Comment by kehh** [comment]
w00t

**Pull request merged**